### PR TITLE
【CINN】Add dsl test cases for dynamic shape schedule

### PR DESCRIPTION
**Pull request by Courtesy-Xs**
### PR types

Others
### PR changes

Others
### Description

pcard-72718
This PR add dsl test cases for dynamic shape schedule

### CODE REVIEW

File: paddle/cinn/pybind/schedule.cc [hunk 1]
```diff
@@ -35,13 +35,18 @@ void BindSchedule(py::module *m) {
...
-            auto scheduler = std::make_unique<ir::IRSchedule>(*module_expr);
-            return scheduler;
-          })
+      .def_static("make",
+                  [](ir::LoweredFunc &ir_func) {
+                    ir::ModuleExpr *module_expr =
+                        new ir::ModuleExpr({ir_func->body});
+                    auto scheduler = std::make_unique<ir::IRSchedule>(
+                        *module_expr,
+                        -1,
```
Comment (zhhsplendid) [inline paddle/cinn/pybind/schedule.cc:44]: Usually, we add some parameter comment to -1, true, false input to increase readability.

For example:
/* param_option = */ -1,

**Review by BiynXu — APPROVED** [review]
LGTM

File: paddle/cinn/pybind/schedule.cc (hunk 1 above)
Comment (Courtesy-Xs) [reply paddle/cinn/pybind/schedule.cc:44]: Ok, I will fix it in next PR.

**Review by zhhsplendid — APPROVED** [review]
LGTM

File: paddle/cinn/pybind/schedule.cc (hunk 1 above)
Comment (Courtesy-Xs) [reply paddle/cinn/pybind/schedule.cc:44]: The PR is [#60342](https://github.com/PaddlePaddle/Paddle/pull/60342)

**Pull request merged**